supervisord.conf

```diff
@@ -30,15 +30,16 @@ command=java -jar %(ENV_GSK_DIST_PATH)s/backend/giskard.jar
 stdout_logfile=%(ENV_GSK_HOME)s/run/%(program_name)s.log
 autorestart=true
 redirect_stderr=true
+stopsignal=INT
 startsecs=5
-command=/bin/bash -c '[ ! -d $PGDATA ] && initdb -D "$PGDATA" -U giskard; postgres -D "$PGDATA"'
+command=/bin/bash -c '[ ! -d $PGDATA ] && initdb -D "$PGDATA" -U giskard; bash $GSK_DIST_PATH/file-guard.sh "$PGDATA/postmaster.pid"; postgres -D "$PGDATA"'
```

Comment:
at this point I think having `start-db.sh` and `start-worker.sh` might be a cleaner solution, WDYT?

Reply:
Sure, let me add them

Reply:
this PR can be related
https://github.com/Giskard-AI/giskard/pull/1398